cmd/faucet/faucet.go

```diff
@@ -178,288 +134,164 @@ func validAddress(addr string) error {
 	return nil
 }
 
-func (c *Client) sendBatchLoop(ctx context.Context) {
-	ticker := time.NewTicker(c.cfg.BatchInterval)
-	for {
-		select {
-		case <-ctx.Done():
-			return
-		case <-ticker.C:
-			if err := c.sendBatchIfNeededLock(ctx); err != nil {
-				log.Printf("error sending batch: %s", err)
-			}
-		}
+func InitFaucet(logger zerolog.Logger) (Faucet, error) {
+	var err error
+	cfg, err := config.LoadConfig()
+	if err != nil {
+		panic(err)
```

Comment:
i think the bot was right though, any reason for panicking here?

Reply:
The idea was that if the configuration is incorrect, there's no reason for the process to start.
I'll change it to Fatal logging.